whois-nrtm4/src/main/java/net/ripe/db/nrtm4/NrtmFileRepo.java

```diff
@@ -0,0 +1,48 @@
+package net.ripe.db.nrtm4;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.OutputStream;
+import java.nio.charset.StandardCharsets;
+
+
+@Service
+public class NrtmFileRepo {
+
```

Comment:
THis is not a repository layer, can we rename it to may be NrtmFileStreamer or something related?

Reply:
it's a repository in the sense that it's a directory for a certain type of file, like a git repo. i'll call it NrtmFileStore

Reply:
methods in this class are more related to FileHelper than related to Nrtm. May be try to move or re-use existing methods in FileHelper in whois-commons?